src/parse_args.ts

```diff
@@ -104,6 +111,20 @@ program
     '--quiet-exit-code',
     'always return 0 as an exit code status, regardless of test pass / fail. Only return > 0 exit codes on internal errors where the suite could not be run'
   )
+  .addOption(
```

Comment:
++ on keeping them separate. JSON makes sense for `--params` because it involves nesting and structure we can't anticipate, but it's a very weird thing to do. Really like the args we have here.